cycle_en_terre_website_design/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Cycle en terre Website design',
+    'category': 'Website',
+    'summary': 'Design modification to look like their own WordPress Website '
+               'http://www.cycle-en-terre.be',
+    'website': 'http://www.cycle-en-terre.be',
+    'author': 'Nicolas Jamoulle <nicolas@coopiteasy.be>',
```

Comment:
For the `author` key, no consensus have be found yet. But I suggest to put "Coop IT Easy SCRLfs" in it. Maybe like this.

```
'author': "Nicolas Jamoulle <nicolas@coopiteasy.be>,"
          "Coop IT Easy SCRLfs",
```